installers/connect-installer/setup.py

```diff
@@ -706,65 +706,101 @@ def codesign_osx(create_dmg=True, notarize=True):
-        'environment variables and the ftrack_connect_sign_pass on the '
-        'keychain before codesign. Also make sure to have appdmg installed '
-        'by running "npm install -g appdmg"',
-    )
-    parser.add_argument(
-        '-cf',
-        '--codesign_frameworks',
-        action='store_true',
-        help='Codesign the frameworks on the Frameworks folder on MacOS',
-    )
+    if sys.platform == 'darwin':
+        parser = argparse.ArgumentParser(
+            prog='setup.py bdist_mac',
+            add_help=True,
+            description=''' Override help for Connect build in MacOs. These are the 
+            accepted arguments for connect build. ''',
+            epilog='Make sure you have the CODESIGN_IDENTITY and APPLE_USER_NAME '
+            'environment variables and the ftrack_connect_sign_pass on the '
+            'keychain before codesign. Also make sure to have appdmg installed '
+            'by running "npm install -g appdmg"',
+        )
+        parser.add_argument(
+            '-cf',
+            '--codesign_frameworks',
+            action='store_true',
+            help='Codesign the frameworks on the Frameworks folder on MacOS',
+        )
+    else:
+        parser = argparse.ArgumentParser(
+            prog='setup.py bdist_msi|build_exe',
+            add_help=True,
+            description=''' Override help for Connect build in Windows. These are the 
+            accepted arguments for connect build. ''',
+            epilog='Make sure you have installed Java & gloud CLI tools and have '
+            'authenticated with Google cloud. ',
+        )
     parser.add_argument(
         '-cs',
         '--codesign',
         action='store_true',
-        help='Codesign the .app in MacOS',
-    )
-    parser.add_argument(
-        '-dmg',
-        '--create_dmg',
-        action='store_true',
-        help='Create the dmg file for MacOS',
-    )
-    parser.add_argument(
-        '-not',
-        '--notarize',
-        action='store_true',
-        help='Notarize the dmg application after codesign',
+        help='Codesign Connect',
     )
+    if sys.platform == 'darwin':
+        parser.add_argument(
+            '-dmg',
+            '--create_dmg',
+            action='store_true',
+            help='Create the dmg file for MacOS',
+        )
+        parser.add_argument(
+            '-not',
+            '--notarize',
+            action='store_true',
+            help='Notarize the dmg application after codesign',
+        )
     args, unknown = parser.parse_known_args()
     sys.argv = [sys.argv[0]] + unknown
-    osx_args = args
 
 
 def clean_download_dir():
     if os.path.exists(DOWNLOAD_PLUGIN_PATH):
         shutil.rmtree(DOWNLOAD_PLUGIN_PATH)
 
 
+def codesign_windows(relative_path):
+    return_code = subprocess.call(
+        ["CMD.EXE", "/C", "codesign.bat", relative_path], shell=True
+    )
+    logging.info(f'Exitcode from code sign: {return_code}')
+
+
+def add_codesign_cx_Freeze():
```

Comment:
maybe add_codesign_cx_freeze_windows? Because this is just windows specific right?
And add docstring here.